BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -349,17 +341,17 @@ Session resumption drastically reduces the number of TLS handshakes and thus is
 performance feature for most deployments.
 
 Stateless session resumption with session tickets is a popular strategy. For TLS 1.2, it is specified in
-{{?RFC5077}}.  For TLS 1.3, an equivalent PSK-based mechanism is described in
-Section 4.6.1 of {{RFC8446}}.
+{{?RFC5077}}.  For TLS 1.3, a more secure PSK-based mechanism is described in
+Section 4.6.1 of {{RFC8446}}. See [this post](https://blog.filippo.io/we-need-to-talk-about-session-tickets/) for a comparison of TLS 1.2 and 1.3 session resumption, and {{Springall16}} for a quantitative study of TLS cryptographic "shortcuts", including session resumption.
+
 When it is used, the resumption information MUST
 be authenticated and encrypted to prevent modification or eavesdropping by an attacker.
 Further recommendations apply to session tickets:
 
 * A strong cipher suite MUST be used when encrypting the ticket (as least as strong as the main TLS cipher suite).
-
-* Ticket keys MUST be changed regularly, e.g., once every week, so as not to negate the benefits of forward secrecy (see {{sec-pfs}} for details on forward secrecy).
-
+* Ticket keys MUST be changed regularly, e.g., once every week, so as not to negate the benefits of forward secrecy (see {{sec-pfs}} for details on forward secrecy). Old ticket keys MUST be destroyed shortly after a new key version is made available.
 * For similar reasons, session ticket validity SHOULD be limited to a reasonable duration (e.g., half as long as ticket key validity).
+* To prevent certain attacks, a TLS 1.2 server SHOULD NOT resume sessions that are too old, e.g. sessions that have been open longer than two ticket key rotation periods. Note that this implies that some server implementations might need to abort sessions after a certain duration.
```

Comment:
We should probably qualify the _certain_ in "certain attacks"

Reply:
> We should probably qualify the _certain_ in "certain attacks"

TLS 1.2 does not roll the session key forward within a single session. Thus, to prevent an attack where a stolen ticket key is used to decrypt the entire content of a session (negating the concept of forward secrecy)...